spring-boot/admin/src/main/java/eu/coatrack/admin/controllers/GatewayApiController.java

```diff
@@ -51,4 +66,26 @@ public ApiKey findApiKeyEntityByApiKeyValue(@RequestParam("keyValue") String api
     public ServiceApi findServiceByApiKeyValue(@RequestParam("apiKeyValue") String apiKeyValue) {
         return serviceApiRepository.findByApiKeyValue(apiKeyValue);
     }
+
+    @GetMapping( "/api/gateways/{gatewayId}/receiveApiKeyList")
+    public ResponseEntity<GatewayUpdate> findServiceByGatewayId(@PathVariable("gatewayId") String gatewayId) {
```

Comment:
This method name is confusing, probably it should have a name like:
`getValidApiKeysForGivenGateway` or `getApiKeysForGateway`?

Reply:
Done.